### DISCUSSION TITLE
upgrade etcd from 3.3 to 3.5 on Ubuntu 22

### DISCUSSION THREAD

**Discussion by angelorso007** [etcd-io/etcd#17824] [Q&A] [answered]
Hello all,
I upgraded from 3.3 to 3.5 on a new cluster following this method:

1 apt install etcd ( installed etcd 3.3 on Ubuntu 22)

2 run the following and update etcd :
          export RELEASE=$(curl -s https://api.github.com/repos/etcd-io/etcd/releases/latest|grep tag_name | cut -d '"' -f 4)
                wget https://github.com/etcd-io/etcd/releases/download/${RELEASE}/etcd-${RELEASE}-linux-amd64.tar.gz
                      tar xvf etcd-${RELEASE}-linux-amd64.tar.gz
                            cd etcd-${RELEASE}-linux-amd64

3 Move **etcd** and **etcdctl**  etcdutl binary files to /usr/local/bin directory 
sudo mv etcd etcdctl etcdutl /usr/local/bin 

On ETCD website I can see differently but from I understand the autor of the article is considering an existing cluster and configuration.

Can you please confirm that what I am doing is correct?

Many thanks
Angelo Onofri

**Comment by jmhbnz** [comment, marked as answer]
Hi @angelorso007 - Thanks for your question. If you are installing a new etcd cluster don't install from `apt` as the version is out of date as you have discovered.

Please just install etcd using the pre built binaries as outlined here: https://etcd.io/docs/v3.5/install.

**Comment by angelorso007** [comment]
Hello

Thank you for your suggestion.
I will follow.

Angelo

Il giorno dom 5 mag 2024 alle ore 11:27 James Blair <
***@***.***> ha scritto:
